frontend/src/components/Layout/VirtualGrid/pipeline.ts

```diff
@@ -320,10 +320,7 @@ export function getVisibleItems<T>(
 
   return allItems.slice(first, last).map((value, localIndex) => {
     const index = first + localIndex;
-    const { x, y } = getItemOffsetByIndex(
-      index,
-      resizeMeasurement
-    ) as ItemOffset;
+    const { x, y } = getItemOffsetByIndex(index, resizeMeasurement);
```

Comment:
@aweebs Good catch!

Do you have any proposal for any linter rules that helps us avoid this kind of missteps in the future.
Essentially every rule that could avoid us having another stylistic PR like this one is welcome